### ISSUE TITLE
Support representing numbers as roman numerals

### ISSUE THREAD

**Issue by mirdaki** [mirdaki/theforce#44]
With inspiration from the Episode numbering, it might be fund to support roman numerals as an option for regular numbers. Their may be a more in-universe syntax or option as well, but I couldn't think of one we could actually enter as text

**Comment by Forgind** [comment]
<img width="348" alt="image" src="https://github.com/mirdaki/theforce/assets/12969783/21e8f488-66be-4575-b5c3-293dc7144f81">

**Comment by mirdaki** [comment]
Yes, aurebesh exists, but it is effectively a different font. As far as I'm aware, there isn't a way to have multiple fonts at the same time in an editor, but I may be wrong. If that is supported and we can specify when fonts change, we can have digits always be aurebesh. The rest of the aurebesh character set could be supported, but the other characters are much harder to infer

**Comment by Forgind** [comment]
Why would you need multiple fonts? You'd have one font that displays most characters 'the normal way' but numerals in aurebesh.

**Comment by mirdaki** [comment]
You could do that. But that means the users preferred font likely doesn't match what the alpha characters are. Maybe that doesn't matter, since it would just be a fun thing to do.

The other thing is that approach is really an editor thing, not a language thing. Not saying we shouldn't support features for that, but the suggestion that inspired this issue was more about getting language to feel that little more Star Wars-y

**Comment by mirdaki** [comment]
Thanks to @GregT2207 we have numerals for the current episode numbers! I'll leave this open to support roman numerals more broadly
